src/Campaigns/Blocks/shared/components/CampaignDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import {PanelBody, SelectControl} from '@wordpress/components';
+import {InspectorControls} from '@wordpress/block-editor';
+import useCampaigns from '../hooks/useCampaigns';
+import {Campaign} from '@givewp/campaigns/admin/components/types';
```

Comment:
Can we remove this not-used import statement?

Reply:
Resolved: 1da92bae836c144549774d5cbbada038a1da6c38